packages/appsync-modelgen-plugin/src/preset.ts

```diff
@@ -49,17 +55,46 @@ const generateJavaPreset = (
         selectedType: modelName,
       },
     });
-  });
```

Comment:
Is moving the below into the forEach loop intentional?

Reply:
No, great catch. I also removed manyToManyJoinModels from the for loop